app/views/users/show.html.erb

```diff
@@ -1,4 +1,30 @@
-<h1>Details of current user</h1>
-<p><strong>Name:</strong> User 1</p>
-<p><strong>Bio:</strong> Description of user 1</p>
-<p><strong>Posts Counter:</strong> 12</p>
\ No newline at end of file
+<div class="user-container">
+  <div class="user-wrapper">
+    <div class="photo-box bordered">
+      <img class="user-img" src="
+        <% @user.photo %>" alt="<%= @user.name %>">
+    </div>
+
+    <div class="text-box bordered">
+      <h3><%= @user.name %></h3>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+
+  <section class="bio bordered">
+    <h2>BIO</h2>
+    <p class="bio-text"><%= @user.bio %></p>
+  </section>
+
+
+  <section>
+    <% @user.posts.each_with_index do |post, index| %>
+      <div class="post bordered">
+        <h3>Post #<%= (index + 1) %></h3>
+        <p><%= post.text %></p>
+        <p class="right">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+      </div>
+    <% end %>
+  </section>
+</div>
```

Comment:
- [OPTIONAL] I made this optional because it was not mentioned by the previous reviewer, but it is highly recommended to have it on the user's show page. It is required that your user's show page contains `See all posts` link that opens user's post index as displayed in the wireframe below. That is yet to be implemented on your user's show page. Kindly attend to it 

![all-posts](https://github.com/M-Anwar-Hussaini/BlogApp/assets/26597432/7af32a54-a793-45aa-9006-dc1a2d52b94a)

-  It should look something similar to this

```
<button>
<%= link_to 'See all Post', user_posts_path(@user) %>
</button>
```